tests/testPosixStorage.py

```diff
@@ -139,7 +139,14 @@ def testAbsolutePath(self):
         f.close()
         del f
         f = storage.getLocalFile('foo.txt')
-        self.assertIsInstance(f, file)
+        try:
+            fileType = file
+        except NameError:
+            from io import IOBase
+            fileType = IOBase
```

Comment:
Can you use use io.IOBase in in Python 2? It is part of the standard library. If it works, I think using it would be a lot cleaner than the above. Just import it and use it.

If that does not work for some reason, then I think this would be significantly cleaner if your new code was put with the other import statements at the top of the file.